controllers/resources/bentorequest_controller.go

```diff
@@ -1737,16 +1737,6 @@ echo "Done"
 		TTY:             true,
 		Stdin:           true,
 		SecurityContext: builderContainerSecurityContext,
-		Resources: corev1.ResourceRequirements{
```

Comment:
This will result in the inability to create image builder Pods in a multi-tenant namespace due to resource quotas present in the multi-tenant namespace.

Reply:
@ssheng is this something you had already thought of?

Reply:
If a namespace has quotas, a pod cannot be created without specifying limits?

Reply:
@ssheng Yes, in a namespace with resource quota, pods must have resources set in order to be created.

Reply:
Understood. We will explicitly add limits from multi-tenant deployment through configmap override. For open source, we can remove the resource limits. Do you think that's reasonable?

Reply:
Sure, we need to add a parameter in the helm values to inform yatai-image-builder whether it is necessary to have default resource settings.